public/icons/site.webmanifest

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "HedgeDoc",
+    "short_name": "HedgeDoc",
+    "icons": [
+        {
+            "src": "/icons/android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "/icons/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#303030",
+    "background_color": "#303030"
```

Comment:
These are no theme colors of the HedgeDoc branding anymore.

Reply:
I changed the theme color, but not not the background color.